src/include/distributed/worker_transaction.h

```diff
@@ -24,7 +24,8 @@ typedef enum TargetWorkerSet
 {
 	NON_COORDINATOR_METADATA_NODES,
 	NON_COORDINATOR_NODES,
-	ALL_SHARD_NODES
+	ALL_SHARD_NODES,
+	METADATA_NODES
```

Comment:
maybe add single line command /* including the coordinator, if exists */ or such

Reply:
I added comments to all the enums